ArduPlane/Plane.h

```diff
@@ -983,6 +986,7 @@ class Plane : public AP_Vehicle {
     void failsafe_long_on_event(enum failsafe_state fstype, ModeReason reason);
     void failsafe_short_off_event(ModeReason reason);
     void failsafe_long_off_event(ModeReason reason);
+    bool in_long_failsafe() const; //returns true if in long_failsafe
```

Comment:
remove function and use variable directly

Reply:
done